client/src/components/nodeRamChart.tsx

```diff
@@ -33,9 +33,12 @@ export default function NodeRamChart({items, metrics}: {items?: Node[], metrics?
 function getNodeRamTotals(items?: Node[], metrics?: _.Dictionary<Metrics>) {
     if (!items || !metrics) return undefined;
 
-    const metricValues = Object.values(metrics);
-    const used = _.sumBy(metricValues, x => parseRam(x.usage.memory));
-    const available = _.sumBy(items, x => parseRam(x.status.capacity.memory));
-
-    return {used, available};
+    try {
```

Comment:
Thanks for your contribution. Instead of adding a try catch here, we will it may be better to check the null cases like above.